api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirObservationServiceImpl.java

```diff
@@ -31,10 +32,11 @@
 import org.springframework.stereotype.Component;
 import org.springframework.transaction.annotation.Transactional;
 
+@Getter(AccessLevel.PROTECTED)
```

Comment:
This should go right above the `@Setter` annotation.

Reply:
O k let me do it now